src/main/java/org/broadinstitute/hellbender/tools/exome/CreatePanelOfNormals.java

```diff
@@ -39,13 +40,38 @@
  * </p>
  *
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ *
+ * <h3>Examples</h3>
+ * <p>
+ *     The command encompasses empirically determined parameters for TCGA project data.
```

Comment:
Nope, Sam doesn't get back until Wednesday and we need to get moving. So I need to go with what Lee is saying.